pywis_pubsub/subscribe.py

```diff
@@ -128,13 +129,16 @@ def subscribe(ctx, config, download, bbox=[], verbosity='NOTSET'):
     config = util.yaml_load(config)
 
     broker = config.get('broker')
+    client_id = config.get('client_id', f'pywis-pubsub-{random.randint(0, 1000)}')
```

Comment:
Can you fix flake8 here?

Reply:
Certainly. If I read build correctly it was

    ./pywis_pubsub/subscribe.py:132:80: E501 line too long (82 > 79 characters)

later it says

    Cannot read properties of undefined (reading 'id')

unsure what that means - I've only fixed above